src/scripts/main.js

```diff
@@ -1,3 +1,116 @@
 'use strict';
 
-// write code here
+const link = `https://mate-academy.github.io/
```

Comment:
it is better to keep only `https://mate-academy.github.io/` in `BASE_LINK` constant as the endpoint may change later